DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateGroupsController.cs

```diff
@@ -47,6 +57,43 @@ public IActionResult GroupDelegates(int groupId, int page = 1)
             return View(model);
         }
 
+        [HttpGet]
+        [Route("{groupId:int}/Delegates/Remove/{delegateId:int}")]
+        public IActionResult GroupDelegatesRemove(int groupId, int delegateId)
+        {
+            var centreId = User.GetCentreId();
+            var groupName = groupsDataService.GetGroupName(groupId, centreId)!;
+            var delegateUser = userDataService.GetDelegateUserById(delegateId)!;

```

Comment:
I'd expect a 404 if the delegate with that id isn't found. In fact, we should give 404 if a delegate is not in that group.